src/Game/GameObjects/Views/MobileView.cs

```diff
@@ -82,8 +82,15 @@ public override bool Draw(UltimaBatcher2D batcher, int posX, int posY)
 
             if (ProfileManager.Current.HighlightGameObjects && SelectedObject.LastObject == this)
             {
-                _viewHue = Constants.HIGHLIGHT_CURRENT_OBJECT_HUE;
-                HueVector.Y = 1;
+                Item item = World.Items.Get(Serial);
+
+                if (this == item)
```

Comment:
I don't see that this will ever be true as this draw is only called for mobiles.

Reply:
Aye, at present it was an oversight brought up in bugs channel. If you paralyze, poison, or stun a mobile, with highlightgamebjects checked this will override highlight by state instead of returning the mobiles highlight color to their notoriety.

Reply:
I'm suggesting that

`if (this == item)`

is always false, and therefore useless.

Reply:
It seemed useful for healthbars, corpses, nametags, but I may have missed others in testing

Reply:
remembering why I added this---(was made on the fly last week)

if mobile is selectedtarget and i hover over healthbar/mobile it will only show the overridden color and not the notoriety of the mobile. If you have a better method to achieve this result instead, please I am all ears.